src/Tests/Microsoft.Diagnostics.Monitoring.EgressExtensibilityApp/Program.cs

```diff
@@ -0,0 +1,83 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Diagnostics.Monitoring.Tool.UnitTests;
+using Microsoft.Diagnostics.Tools.Monitor.Egress;
+using System;
+using System.Collections.Generic;
+using System.CommandLine;
+using System.Text.Json;
+
+namespace Microsoft.Diagnostics.Monitoring.EgressExtensibilityApp
+{
+    internal class Program
+    {
+        static int Main(string[] args)
+        {
+            RootCommand rootCommand = new RootCommand();
+
+            Command egressCmd = new Command("Egress");
+
+            egressCmd.SetHandler(() => Egress());
+
+            rootCommand.Add(egressCmd);
+
+            return rootCommand.Invoke(args);
+        }
+
+        private static int Egress()
+        {
+            EgressArtifactResult result = new();
+            try
+            {
+                string jsonConfig = Console.ReadLine();
```

Comment:
Not directly related to your changes, but it would be great if we had a test and example of how we want extensibility writers to consume configuration and artifacts.